test/integration/event_log_test.rb

```diff
@@ -237,6 +237,34 @@ class EventLogIntegrationTest < ActionDispatch::IntegrationTest
     assert page.has_content?("You do not have permission to perform this action")
   end
 
+  test "record user's login ip address" do
+    visit root_path
+    signin_with(@user)
+
+    ip_address = @user.event_logs.last.trailing_message.split.first
+    assert ip_address.match(/\A[[:alnum:].:]+$/)
+  end
+
+  test "record user's login browser" do
+    visit root_path
+    signin_with(@user)
+
+    browser = @user.event_logs.last.trailing_message.split.last
+    assert browser.match(/\A[[:alpha:]]+/)
+  end
+
+  test "record who the account was created by" do
+    visit root_path
+    signin_with(@admin)
+    visit users_path
+    click_on "Create user"
+    fill_in 'Name', with: 'New User'
+    fill_in 'Email', with: 'test@test.com'
+    click_on "Create user and send email"
+
+    assert_equal EventLog::ACCOUNT_INVITED, User.last.event_logs.first.entry
```

Comment:
Could we also assert that the event log created has the`@admin` user associated with it as the initiator, not just that we've got an `ACCOUNT_INVITED` event.  Knowing who invited them is one of the key things we want to know after all, so we should check it if we can.